desktop/ui/src/main/java/org/datacleaner/extensionswap/ExtensionSwapClient.java

```diff
@@ -23,7 +23,8 @@
 import java.util.List;
 
 import org.apache.commons.vfs2.FileObject;
-import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.CloseableHttpClient;
```

Comment:
Using an impl package from outside the project usually worries me, most projects has internal classes in there. Are you sure this is being used right?

Of course, it might just be the Http project that doesn't follow that :)